### ISSUE TITLE
error reading in OME-TIFF

### ISSUE THREAD

**Issue by emmanuel-contreras** [cgohlke/tifffile#72]
Hello, I recently upgraded to the latest version and noticed that I was no longer able to load our OME-TIFF images, getting the error below. I downgraded to 2020.10.01 (using conda) and the error disappeared. Not sure if this is a problem with our images or an update on the module. 

Do you know why I may be getting this error on the latest version?  

```
IPdb [3]: im = tifffile.imread(path_image)
OME series: failed to read 'H9_mitotracker-060_Cycle00001_Ch3_000001.ome.tif'
OME series: expected 2 frames, got 1
*** ValueError: failed to read 4194304 bytes
```

Thank you

**Comment by cgohlke** [comment]
Can you share the file?

**Comment by emmanuel-contreras** [comment]
Sure, here it is. I had to change the extension from ".tif" to ".png" otherwise github wouldn't let me upload it. Thank you for taking a look at it, please let me know if you need more info.

![H9-mitotracker-060-Cycle00001-Ch3-000001 ome](https://user-images.githubusercontent.com/14666496/111502828-34cedd00-8714-11eb-8e69-1e8c459afe8b.png)

**Comment by cgohlke** [comment]
The file is part of a series of files. Can you share the whole series, especially the MetadataFile `H9_mitotracker-060_Cycle00001_Ch1_000001.ome.tif`?

**Comment by emmanuel-contreras** [comment]
Sure, here it is
![H9_mitotracker-060_Cycle00001_Ch1_000001 ome](https://user-images.githubusercontent.com/14666496/111504999-297cb100-8716-11eb-88ce-4df6d4165279.png)

**Comment by cgohlke** [comment]
I can reproduce this issue. It happens when the second file `'H9_mitotracker-060_Cycle00001_Ch3_000001.ome.tif'` is not present. This case is actually tested, but the test case was not well chosen...

**Comment by emmanuel-contreras** [comment]
ahh ok, sounds good. It also sounds like I've been loading the series files incorrectly... I usually just load one of the files, I see a brief error but it still loads the individual image correctly. Thanks for the help!

**Comment by cgohlke** [comment]
Thanks for reporting the issue. Should be fixed in v2021.3.17. Although it might be better to raise an error if files are missing from an OME-TIFF series...
